Automattic/jetpack
# Contact Form: avoid PHP notices in submitted forms in some cases.

Fixes #22079

#### Changes proposed in this Pull Request:

* Avoid a PHP notice in some edge-cases with the Contact form module.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

I am not quite sure how to reproduce this, so I do not have detailed steps to test for that specific bug.

* Go to Jetpack > Settings and ensure that the contact form module is active.
* Go to Posts > Add New and create a new form block, with multiple different fields.
* Publish post, and submit a test form.
* Check the email you got; it should inlude all the fields you've submitted.
